MapboxNavigation/NavigationViewController.swift

```diff
@@ -707,7 +707,7 @@ extension NavigationViewController: TopBannerViewControllerDelegate {
         guard let stepIndex = leg.steps.firstIndex(of: step) else { return }
         let nextStepIndex = stepIndex + 1
         
-        let legProgress = RouteLegProgress(leg: leg, stepIndex: stepIndex)
+        let legProgress = try! RouteLegProgress(leg: leg, stepIndex: stepIndex)
```

Comment:
Same concern about force unwrapping here.
Actually we do the crash in the worst-case scenario. Probably that's ok, but can we try avoid this?